src/components/tooltip/Tooltip.stories.mdx

```diff
@@ -40,17 +40,17 @@ import { Input } from "../input";
 
 <Canvas>
   <Story name="bottom">
-    <Flex
+    <FlexContainer
       justifyContent="space-evenly"
-      style={{ paddingTop: "3rem", paddingBottom: "3rem" }}
+      css={{ paddingTop: "3rem", paddingBottom: "3rem" }}
```

Comment:
I've used it just to be more consistent with using styled-components. Also in our guidelines we decided not to use style attribute because it's lacking some css features: https://github.com/nordcloud/pat-frontend-docs/blob/master/react/STYLING.md#inline-styles

Reply:
cool! much thanks for linking the docs 🙏